tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_in_memory_data_using_pandas.py

```diff
@@ -0,0 +1,41 @@
+"""
+To run this code as a local test, use the following console command:
+```
+pytest -v --docs-tests -m integration -k "how_to_connect_to_in_memory_data_using_pandas" tests/integration/test_script_runner.py
+```
+"""
+
+import pathlib
+import great_expectations as gx
+
+context = gx.get_context()
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_in_memory_data_using_pandas.py datasource">
+datasource = context.sources.add_pandas(name="my_pandas_datasource")
+# </snippet>
+
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_in_memory_data_using_pandas.py dataframe">
+import pandas as pd
+
+dataframe = pd.read_parquet(
+    "https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-11.parquet"
+)
+# </snippet>
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_in_memory_data_using_pandas.py name">
+name = "taxi_dataframe"
+# </snippet>
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_in_memory_data_using_pandas.py data_asset">
+data_asset = datasource.add_dataframe_asset(name=name, dataframe=dataframe)
```

Comment:
@TrangPham This is old API signature -- as per #8032 (merged yesterday before the release), we now only specify the name and metadata in `add_dataframe_asset()` -- and the actual `DataFrame` is specified in `build_batch_request()`.  Thank you.